docs/config_topdown_centered_instance.yaml

```diff
@@ -1,51 +1,39 @@
 data_config:
   provider: LabelsReader
-  pipeline: TopdownConfmaps
-  train:
-    labels_path: "minimal_instance.pkg.slp"
+  train_labels_path: minimal_instance.pkg.slp
+  val_labels_path: minimal_instance.pkg.slp
+  preprocessing:
     max_width: 
     max_height: 
     scale: 1.0
     is_rgb: False
-    preprocessing:
-      crop_hw:
+    crop_hw:
       - 160
       - 160
-    use_augmentations: true
-    augmentation_config:
-      geometric:
-        rotation: 180.0
-        scale: 0
-        translate_width: 0
-        translate_height: 0
-        affine_p: 0.5
-  val:
-    labels_path: "minimal_instance.pkg.slp"
-    max_width: 
-    max_height: 
-    is_rgb: False
-    scale: 1.0
-    preprocessing:
-      crop_hw:
-      - 160
-      - 160
-    use_augmentations: false
+  use_augmentations_train: true
+  augmentation_config:
+    geometric:
+      rotation: 180.0
+      scale: 0
+      translate_width: 0
+      translate_height: 0
```

Comment:
Are these interpreted as "null" / `None`? Scale of 0 might lead to some funky scaling...

Reply:
yes, will change them to `null`s